dgraphtest/dgraph.go

```diff
@@ -116,7 +116,8 @@ func (z *zero) ports() nat.PortSet {
 func (z *zero) cmd(c *LocalCluster) []string {
 	zcmd := []string{"/gobin/dgraph", "zero", fmt.Sprintf("--my=%s:%v", z.aname(), zeroGrpcPort), "--bindall",
 		fmt.Sprintf(`--replicas=%v`, c.conf.replicas), fmt.Sprintf(`--raft=idx=%v`, z.id+1), "--logtostderr",
-		fmt.Sprintf("-v=%d", c.conf.verbosity)}
```

Comment:
Why did you remove this flag?

Reply:
Yeah, we shouldn't. @jbhamra1 please check. Do you want me to check?